[RHACS] ROX15257 ACSCS service description documentation (#59140)
openshift/openshift-docs: architecture/rhacs-cloud-service-service-description.adoc

:_content-type: ASSEMBLY
[id="rhacs-cloud-service-service-description"]
= {product-title-managed-short} service description
include::modules/common-attributes.adoc[]
:context: access-policy-service-description

toc::[]

[id="introduction-to-rhacs_{context}"]

> == Introduction to {product-title-short}

Author: @kcarmichael08 yes we can, but for those headings with xref, we need to leave it as is since we can use xrefs only within an assembly.

> == Introduction to {product-title-short}

Reviewer: Yeah, I think with how the links in those sections need to be incorporated into the text, there isn't another way we can do these sections as modules.

> == Introduction to {product-title-short}

Reviewer: Yeah, I think with how the links in those sections need to be incorporated into the text, there isn't another way we can do these sections as modules.

{rh-rhacscs-first} provides Kubernetes-native security as a service.
With {product-title-managed-short}, Red Hat maintains, upgrades, and manages your Central services.

Central services include the user interface (UI), data storage, {product-title-short} application programming interface (API), and image scanning capabilities.
You deploy your Central service through the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console.] When you create a new Central instance, Red Hat creates your individual control plane for {product-title-short}.

{product-title-managed-short} allows you to secure self-managed clusters that can reach out to a Central instance.
The clusters you secure, called Secured Clusters, are managed by you, and not by Red Hat.
Secured Cluster services include optional vulnerability scanning services, admission control services, and data collection services used for runtime monitoring and compliance.
You install Secured Cluster services on any OpenShift or Kubernetes cluster you want to secure.

[id="billing_{context}"]
== Billing

Customers can purchase a {product-title-managed-short} subscription on the Amazon Web Services (AWS) marketplace.

> The service cost is charged hourly per secured core ^[1]^.

Reviewer: ```suggestion
The service cost is charged hourly per secured core, or vCPU of a node belonging to a secured cluster.
```

.Example
====
If you have established a connection to two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).
====

[.small]
--
[1] vCPU of a node belonging to a secured cluster.
--

[id="security-and-compliance_{context}"]
== Security and Compliance

All {rh-rhacscs-first} data in the Central instance is encrypted in transit and at rest.
The data is stored in secure storage with full replication and high availability together with regularly scheduled backups.
The {product-title-managed-short} is available through cloud data centers that ensure optimal performance and the ability to meet data residency requirements.

[id="authentication-provider_{context}"]
=== Authentication provider

> When you create a Central instance using  link:https://console.redhat.com/[Red Hat Hybrid Cloud Console], authentication for the cluster administrator is configured as part of the process.

Reviewer: ```suggestion
When you create a Central instance using link:https://console.redhat.com/[Red Hat Hybrid Cloud Console], authentication for the cluster administrator is configured as part of the process.
```

Customers must manage all access to the Central instance as part of their integrated solution.
For more information about the available authentication methods, see xref:../operating/manage-user-access/understanding-authentication-providers.adoc#understanding-authentication-providers[Understanding authentication providers].

The default identity provider in {product-title-managed-short} is Red Hat Single Sign-On (SSO). For more information about authentication using Red Hat SSO, see xref:../installing/getting-started-rhacs-cloud-ocp.adoc#default-access-acs-console[Default access to ACS Console].

[id="regulatory-compliance_{context}"]
=== Regulatory compliance

For the latest regulatory compliance information, see link:https://docs.openshift.com/dedicated/osd_architecture/osd_policy/policy-process-security.html[Understanding process and security for OpenShift Dedicated].

[id="metrics-and-logging_{context}"]
== Metrics and Logging

[id="service-metrics_{context}"]
=== Service metrics

Service metrics are internal only.

> Red Hat provides and maintains the service at the agreed level.

Reviewer: agreed = agreed upon? contracted? this phrasing seems odd but maybe this came from Legal.

[id="customer-metrics_{context}"]
=== Customer metrics

Core usage capacity metrics are available either through link:https://access.redhat.com/articles/subscription-watch[Subscription Watch] or the link:https://console.redhat.com/beta/application-services/subscriptions/acs[Subscriptions page].

[id="service-logging_{context}"]
=== Service logging

System logs for all components of the {rh-rhacscs-first} are internal and available only to Red Hat personnel.
Red Hat does not provide user access to component logs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].

[id="scalability-and-service-level_{context}"]
== Scalability and Service Level

{rh-rhacscs-first} has set limits on the number of cores or clusters it can protect.
The limits are based on the resources available in the Secured Cluster and the limits on usability.

[id="service-level-objective-and-agreement_{context}"]
=== Service Level Objective and Agreement

> For more information about Service Level Objectives (SLOs) and Service Level Agreements (SLAs) in link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].

Reviewer: ```suggestion
For more information about Service Level Objectives (SLOs) and Service Level Agreements (SLAs), see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
```

Customers have no control over when a Central service update occurs. For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].
Upgrades to the version of {rh-rhacscs-first} are considered part of the service update.

The version of {product-title-managed-short} used on Secured Clusters must match the version of the Central instance of {product-title-managed-short} to ensure compatibility.
Customers are responsible for Secured Cluster services upgrades required to maintain this version compatibility.

[id="availability_{context}"]
== Availability

Availability and disaster avoidance are extremely important aspects of any security platform. {rh-rhacscs-first} provides numerous protections against failures at multiple levels.
To account for possible cloud provider failures, Red Hat established multiple availability zones.

[id="backup-and-disaster-recovery_{context}"]
=== Backup and Disaster Recovery

All {product-title-managed-short} clusters are backed up using Database backups.
This also applies to customer data stored in the Central database.

> All snapshots are created using the appropriate cloud provider snapshot APIs, encrypted  and then uploaded to secure object storage, which for  Amazon Web Services (AWS) is an S3 bucket.

Reviewer: ```suggestion
All snapshots are created using the appropriate cloud provider snapshot APIs, encrypted  and then uploaded to secure object storage, which for Amazon Web Services (AWS) is an S3 bucket.
```

* Red Hat does not commit to a Recovery Point Objective (RPO) or Recovery Time Objective (RTO). For more information, see link:https://www.redhat.com/licenses/Appendix_4_Red_Hat_Online_Services_20221213.pdf[PRODUCT APPENDIX 4 RED HAT ONLINE SERVICES].

> * Site Reliability Engineering (SRE) performs backups only as a precautionary measure. They are stored in the same region as the cluster.

Reviewer: later "SRE" is identified as site reliability engineer(s) so maybe remove "(SRE)" here

> * Customers should deploy multiple Availability Zone Secured Clusters with workloads that follow Kubernetes best practices to ensure high availability within a region.

Reviewer: ```suggestion
* Customers should deploy multiple availability zone Secured Clusters with workloads that follow Kubernetes best practices to ensure high availability within a region.
```

[id="getting-support_{context}"]
=== Getting Support

The {product-title-managed-short} includes Red Hat Standard and Premium support, which you can access using the link:https://access.redhat.com/support[Red Hat Customer Portal].

> You can raise support tickets for the product "Red Hat Advanced Cluster Security Cloud Service".

Reviewer: ```suggestion
You can open support tickets for the product "Red Hat Advanced Cluster Security Cloud Service".
```

With {product-title-managed-short}, Red Hat maintains, upgrades, and manages your Central services.

Central services include the user interface (UI), data storage, {product-title-short} application programming interface (API), and image scanning capabilities.
You deploy your Central service through the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console.] When you create a new Central instance, Red Hat creates your individual control plane for {product-title-short}.

{product-title-managed-short} allows you to secure self-managed clusters that can reach out to a Central instance.
The clusters you secure, called Secured Clusters, are managed by you, and not by Red Hat.
Secured Cluster services include optional vulnerability scanning services, admission control services, and data collection services used for runtime monitoring and compliance.
You install Secured Cluster services on any OpenShift or Kubernetes cluster you want to secure.

[id="billing_{context}"]
== Billing

Customers can purchase a {product-title-managed-short} subscription on the Amazon Web Services (AWS) marketplace.
The service cost is charged hourly per secured core ^[1]^.

.Example
====

> If you have established a connection between two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).

Reviewer 2: ```suggestion
If you have established a connection to two secured clusters, each with 5 identical nodes with 8 vCPUs (such as Amazon EC2 m7g.2xlarge), the total number of secured cores is 80 (2 x 5 x 8 = 80).
```

Context: in this case, both secured clusters are connected to the ACSCS Central.
